projects/demo-integrations/src/tests/kit/date/date-strict-mode.cy.ts

```diff
@@ -0,0 +1,86 @@
+import {DemoPath} from '@demo/constants';
+
+describe('Date | Strict mode', () => {
+    describe('Disabled', () => {
```

Comment:
Why do all specs contain only tests with `strict=false` ? Probably, we should add tests for `strict=true` too

Reply:
I'll add. I thought all other tests use strict mode^ so we don't need those here